Source/Actors/Actor.cs

```diff
@@ -118,8 +132,7 @@ public virtual void ValidateTransformations()
 		dirty = false;
 
 		matrix =
-			Matrix.CreateRotationZ(facing.XY().Angle() + MathF.PI / 2) *
-			Matrix.CreateRotationY(Facing.Z) *
+			Matrix.CreateFromYawPitchRoll(tilt.X, tilt.Y, facing.Angle() + MathF.PI / 2) *
```

Comment:
I think the x and y here might be backwards of what it should be. It's a bit confusing because I think Matrix uses Assumes Y Up, but Celeste 64 assumes Z up, so yaw and roll are kind of flipped of what you'd expect. But pitch should always be represented by X. I think it should be:
`Matrix.CreateFromYawPitchRoll(tilt.Y, tilt.X, facing.Angle() + MathF.PI / 2)`
I tested it in game with your changes and saw the same result. If I rotated it around what becomes tilt.X, it was rotating around the roll, but tilt.X should probably reflect the pitch.